lib/pure/asynchttpserver.nim

```diff
@@ -173,17 +190,9 @@ proc processRequest(
   for linePart in lineFut.mget.split(' '):
     case i
     of 0:
-      case linePart
```

Comment:
I liked the inline ``case`` statement better.

Reply:
I didn't touch this part of the library. Maybe Dominik Picheta can see this part.

Reply:
I think you based your patch on something else but ``nim devel``?

Reply:
Yes, I based my patch on nim devel.